madmom/features/beats.py

```diff
@@ -1238,6 +1238,380 @@ def add_arguments(parser, min_bpm=MIN_BPM, max_bpm=MAX_BPM,
         return g
 
 
+class MultiAgentBeatTrackingProcessor(OnlineProcessor):
+    """
+    Beat Tracking via Multiple Agents. Oriented towards the paper "Beat
+    Tracking for multiple applications: A multi-agent system architecture with
+    state recovery" by Lobate Oliveira et al., 2012
+
+    Parameters
+    ----------
+    max_agents : int
+        Max number of agents.
+    tempi : int
```

Comment:
To be in line with other beat trackers, this should be `num_tempi`.

Additionally, all these parameters should be available in `__init__`, otherwise they cannot be controlled. If they are optional (i.e. initialised with default values) the should be marked accordingly.